signed/interface.go

```diff
@@ -0,0 +1,20 @@
+package signed
+
+import (
+	"github.com/endophage/go-tuf/data"
+	"github.com/endophage/go-tuf/keys"
+)
+
+type SigningService interface {
+	Sign(keyIDs []string, data []byte) ([]data.Signature, error)
```

Comment:
Yes. It's probably not necessary right now but if we ever add thresholds or some other use case comes up it'll be nice to have and we won't have to update the API.